test/rfc7540_SUITE.erl

```diff
@@ -3121,8 +3121,7 @@ lingering_data_counts_toward_connection_window(Config0) ->
 	doc("DATA frames received after sending RST_STREAM must be counted "
 		"toward the connection flow-control window. (RFC7540 5.1)"),
 	Config = cowboy_test:init_http(?FUNCTION_NAME, #{
-		env => #{dispatch => cowboy_router:compile(init_routes(Config0))},
-		initial_connection_window_size => 100000
```

Comment:
Why change the `initial_connection_window_size` value here? Just curious, if tests pass I have no problem with it.

Reply:
I'm fixing this test and found it's not necessary to set custom initial_connection_window_size here. And we need creating a larger binary for tests.